src/autoscaler/eventgenerator/client/log_cache_client.go

```diff
@@ -24,18 +26,20 @@ type LogCacheClient struct {
 	logger lager.Logger
 	Client LogCacheClientReader
 
-	now               func() time.Time
-	envelopeProcessor envelopeprocessor.EnvelopeProcessor
-	goLogCache        GoLogCache
-	TLSConfig         *tls.Config
-	uaaCreds          models.UAACreds
-	url               string
+	now                func() time.Time
+	envelopeProcessor  envelopeprocessor.EnvelopeProcessor
+	goLogCache         GoLogCache
+	TLSConfig          *tls.Config
+	uaaCreds           models.UAACreds
+	url                string
+	collectionInterval time.Duration
```

Comment:
Added `collectionInterval` to the client after removing it https://github.com/cloudfoundry/app-autoscaler-release/pull/2890/files#r1579368032 since it's required for the PromQL API calls